datagen/src/main.rs

```diff
@@ -181,7 +181,7 @@ fn get_course_info(course: &String, term: &String) -> Result<Course, Box<dyn Err
 
         // Identify sections for the given course
         let sections_selector = create_selector!(format!("#{} .section-id", course));
-        let re = Regex::new(r"[0-9]+").unwrap();
+        let re = Regex::new(r"[A-Z]?[0-9]+").unwrap();
```

Comment:
Possibly make this a Kleene operator instead of `?` to fix issues?

Reply:
Done.